### DISCUSSION TITLE
Change value of work time entry after a period has been approved

### DISCUSSION THREAD

**Discussion by klausjettkant** [kimai/kimai#4600] [General]
When I am billing my costumers, I use exports to prove the delivery of accomplished activities. All the work that has been done is written down in the description field of a work time entry. I want to correct a spelling error of one of my employees, before the export is sent to my customer, but I get the error message: "This period has already been approved, you cannot edit or create new entries before 01.01.24." How can i edit a description entry when it is this far in the past? Thank you in advance.

**Comment by kevinpapst** [comment]
Please post here: https://github.com/kevinpapst/kimai-plugins
